misc/ballerinalang-data-mapper/src/test/resources/codeaction/source/dataMapper10.bal

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
Update the copyright year to 2021

Reply:
Have to do this to all the newly added files this year

Reply:
Changed the copyright year to 2021